tests/wallet/cat_wallet/test_cat_wallet.py

```diff
@@ -827,3 +839,164 @@ async def check_wallets(node):
 
         await time_out_assert(20, cat_wallet.get_confirmed_balance, 35)
         await time_out_assert(20, cat_wallet.get_unconfirmed_balance, 35)
+
+    @pytest.mark.parametrize(
+        "trusted",
+        [True, False],
+    )
+    @pytest.mark.asyncio
+    async def test_cat_change_detection(
+        self, self_hostname: str, two_wallet_nodes_services: SimulatorsAndWalletsServices, trusted: bool
```

Comment:
```suggestion
        self, self_hostname: str, one_wallet_and_one_simulator_services: SimulatorsAndWalletsServices, trusted: bool
```

Not sure if you missed that because i suggested it together with something else but since we don't need to wallets here the fixture should change imo.

Reply:
Changed